app/Models/Forum/Post.php

```diff
@@ -223,6 +223,10 @@ public function isValid()
             $this->validationErrors()->add('post_text', 'required');
         }
 
+        if ($this->isDirty('post_text') && mb_strlen($this->post_text) > config('osu.forum.max_post_length')) {
```

Comment:
This should check `body_raw` length otherwise this will include length of the bbcode uid and other additional attributes.